tests/main/views/test_audits.py

```diff
@@ -687,6 +709,31 @@ def test_should_not_get_audit_event_for_date_with_no_events(self):
         assert response.status_code == 200
         assert len(data['auditEvents']) == 0
 
+    @pytest.mark.parametrize(
+        ("date_range_query", "number_of_events"),
+        (
+            ("<=2000-12-31", 4),
+            (">=2000-01-01", 4),
+            ("<2000-12-31", 3),
+            (">2000-01-01", 3),
+            ("2000-01-02..2000-12-30", 2),
+        ))
+    def test_can_find_audit_events_in_date_range(self, date_range_query, number_of_events):
+        AuditEvent.query.delete()
+
+        self.add_audit_event(created_at=datetime(2000, 1, 1))
+        self.add_audit_event(created_at=datetime(2000, 1, 2))
+        self.add_audit_event(created_at=datetime(2000, 2, 5))
+        self.add_audit_event(created_at=datetime(2000, 12, 31))
+
+        response = self.client.get(
+            "/audit-events",
+            query_string={"audit-date": date_range_query},
+        )
+        assert response.status_code == 200
+        events = response.json["auditEvents"]
+        assert len(events) == number_of_events
+
     def test_should_reject_invalid_audit_dates(self):
```

Comment:
Can we add some parametrized cases to this test as well e.g. `"<=invalid"`? Either here or in the `test_it_should_raise_value_error_if_date_string_is_not_valid` test below

Reply:
Done